PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/actions/CreateArmStorageAccountAction.java

```diff
@@ -26,8 +40,37 @@ public CreateArmStorageAccountAction(StorageModule storageModule) {
     public void actionPerformed(NodeActionEvent e) {
         SignInCommandHandler.requireSignedIn(PluginUtil.getParentShell(), () -> {
             CreateArmStorageAccountForm createStorageAccountForm = new CreateArmStorageAccountForm(PluginUtil.getParentShell(), null, null);
-            createStorageAccountForm.setOnCreate(() -> storageModule.load(false));
+            createStorageAccountForm.setOnCreate(() -> {
+                AzureTaskManager.getInstance().runInBackground(
+                        "Creating storage account " + createStorageAccountForm.getStorageAccount().getName() + "...", new Runnable() {
+                            @Override
+                            public void run() {
+                                EventUtil.executeWithLog(STORAGE, CREATE_STORAGE_ACCOUNT, (operation) -> {
+                                    createStorageAccount(createStorageAccountForm.getStorageAccount());
+                                            storageModule.load(false);
+                                        }, (e) ->
+                                                AzureTaskManager.getInstance().runLater(() ->
+                                                        PluginUtil.displayErrorDialog(PluginUtil.getParentShell(), Messages.err,
+                                                                "An error occurred while creating the storage account: " + e.getMessage())
```

Comment:
use AzureMessager.

Reply:
shall not related to track2. move to future PR